src/dfastbe/io.py

```diff
@@ -1645,6 +1693,35 @@ def mask(
 
         return search_lines, max_distance
 
+    @staticmethod
+    def _select_closest_part(
+        multiline: MultiLineString,
```

Comment:
can you rename this `multiline` to a more descriptive name of the values represent, rather than the type of variable?